modules/weko-records/weko_records/api.py

```diff
@@ -321,8 +321,9 @@ def create(cls, item_type_name=None, name=None, schema=None, form=None,
         return record
 
     @classmethod
-    def update(cls, id_=0, name=None, schema=None, form=None, render=None,
-               tag=1):
+    def create_or_update(cls, id_=0, name=None, schema=None, form=None,
```

Comment:
I have checked by renaming this function and to detect whether there is any function calling this function(If yes, it will raise an error because I have changed the name of the function), but I found no one, so it means this function is only called form 
"modules/weko-itemtypes-ui/weko_itemtypes_ui/admin.py"